src/js/edu-benefits/utils/helpers.js

```diff
@@ -3,7 +3,7 @@ import moment from 'moment';
 
 export const chapterNames = {
   veteranInformation: 'Veteran Information',
-  benefitsEligibility: 'Benefits Eligibility',
+  benefitsEligibility: 'Benefits',
```

Comment:
Has two sub-sections: Benefit Selection and Benefit Relinquishment. Benefits is the union.